### ISSUE TITLE
Update error (AssertionError: can't parse the given html)

### ISSUE THREAD

**Issue by tib-kpl** [maxisoft/Freenom-dns-updater#18]
Hello,

I have a new error when I use fdu.
Can you help me ?

```
Traceback (most recent call last):
  File "/usr/local/bin/fdu", line 11, in <module>
    sys.exit(cli())
  File "/home/tib-kpl/.local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/tib-kpl/.local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/tib-kpl/.local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tib-kpl/.local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tib-kpl/.local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/scripts/fdu.py", line 227, in update
    return _update(config, ignore_errors)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/scripts/fdu.py", line 199, in _update
    ok_count, err_count = record_action(lambda freenom, rec: freenom.add_record(rec, True), config, ignore_errors)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/scripts/fdu.py", line 252, in record_action
    action(freenom, rec)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/scripts/fdu.py", line 199, in <lambda>
    ok_count, err_count = record_action(lambda freenom, rec: freenom.add_record(rec, True), config, ignore_errors)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/freenom.py", line 61, in add_record
    records = self.list_records(record.domain)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/freenom.py", line 53, in list_records
    ret = RecordParser.parse(r.text)
  File "/usr/local/lib/python2.7/dist-packages/freenom_dns_updater/record_parser.py", line 13, in parse
    assert tag, "can't parse the given html"
AssertionError: can't parse the given html
```

**Comment by J-Cake** [comment]
I got this error when I tried [@Shihira's answer](https://github.com/maxisoft/Freenom-dns-updater/issues/11#issuecomment-345633463)

**Comment by maxisoft** [comment]
Should be fixed by #23
